test/e2e/frontend/cypress/tests/team/audit-log.spec.js

```diff
@@ -0,0 +1,20 @@
+describe('FlowForge - Team Audit Log', () => {
+    // let team // , project
+
+    beforeEach(() => {
+        cy.login('alice', 'aaPassword')
+        cy.home()
+    })
+
+    it('should show a placeholder if no activity is present', () => {
+        cy.intercept('/api/*/teams/*/audit-log', {
+            log: [],
+            meta: {}
+        }).as('getAuditLog')
+
+        cy.visit('team/ateam/audit-log')
+        cy.wait(['@getAuditLog'])
+
+        cy.get('.ff-no-data').should('exist')
+    })
+})
```

Comment:
Are you adding test coverage of the accordion behaving? Or is this elsewhere?

Reply:
good point - will put it in